src/classic/element/__tests__/ReactElementValidator-test.js

```diff
@@ -135,7 +141,7 @@ describe('ReactElementValidator', function() {
     spyOn(console, 'warn');
     var Component = React.createFactory(ComponentClass);
 
-    Component(null, { 1: Component(), 2: Component() });
+    Component(null, frag({ 1: Component(), 2: Component() }));
```

Comment:
It is a placeholder as an upgrade path, not a permanent API, which is why it is in addons. We're thinking something like:

`<frag><Component /><Component /></frag>`

or E4X's

`<><Component /><Component /></>`

Or when you want to key an unknown set:

`<div><frag key="a">{childrenA}</frag><frag key="b">{childrenB}</frag></div>`

`createFragment` is just a way to accept legacy object literals as an easy upgrade path.